purchase_discount/__manifest__.py

```diff
@@ -4,20 +4,22 @@
 # © 2016 ACSONE SA/NV (<http://acsone.eu>)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 {
-    "name": "Purchase order lines with discounts",
+    "name": "Purchase Order Lines with Discounts",
     "author": "Tiny, "
               "Acysos S.L., "
               "Tecnativa, "
-              "ACSONE SA/NV,"
+              "ACSONE SA/NV, "
               "Odoo Community Association (OCA)",
-    "version": "11.0.1.0.1",
+    "version": "12.0.1.0.0",
+    "license": "AGPL-3",
```

Comment:
Why do you change the order of the keys? It's better to just reduce the diff to minimum